### PR TITLE
Update sky.com.channels.xml

### PR DESCRIPTION
**Pull request by Smark223**
Add Premier Sports 1 and 2

### CODE REVIEW

**Comment by RevGear** [comment]
Premier Sports 1 and 2 don't exist anymore. They were rebranded as ViaPlay Sport 1 and 2 last year. Some of their coverage still includes the Premier Sports logo, especially the older footage.

**Comment by Smark223** [comment]
They do exist in Ireland https://www.nowtv.com/ie/sports-overview# however, I think you're right in that the site IDs I've used in this amendment are the ones for the old UK Premier Sports channels that are indeed defunct, so this probably needs rejecting. Can someone check?

**Comment by Smark223** [comment]
> I hadn't realised they were still using Premier Sports branding in Ireland. According to Wikipedia, the Irish feed has reverted back to Premier Sports with a slightly different schedule.
> 
> > Viaplay rebranded Premier Sports under the Viaplay Sports name on 1 November 2022. Initially, the same feeds were used for the UK and Ireland with Premier Sports branding used for Premier League coverage which is blocked in the UK. On 28 November the channels split, with the Irish feeds reverting to full Premier Sports branding.
> 
> Maybe we need two new IDs for PremierSportsIreland1.ie and PremierSportsIreland2.ie as these are effectively new channels.

@freearhey created them in the database earlier under https://github.com/iptv-org/database/commit/ef79ea7d0faefc1011cb5fe8afb58ee65f3d7545 but as PremierSports1.ie and PremierSports2.ie

**Comment by Smark223** [comment]
Anyway, I think I've messed up with this request. I'm not able to check if those site IDs are correct at the moment but I would expect the schedules for the Premier Sports channels to be included on the Sky site. I can't get it to show me Irish channels though, even with a VPN.

**Comment by RevGear** [comment]
> @freearhey created them in the database earlier under [iptv-org/database@ef79ea7](https://github.com/iptv-org/database/commit/ef79ea7d0faefc1011cb5fe8afb58ee65f3d7545) but as PremierSports1.ie and PremierSports2.ie Not sure if the 'Ireland' is needed in the ID - unsure what the standard is. It seems when there are multiple versions of a channel with the same name in different countries, sometimes a country suffix is added to the ID but sometime it's not.

I thought PremierSports1.ie and PremierSports2.ie were already in the database, as the original channels prior to ViaPlay branding. Maybe they got deleted instead of marked as closed. I suggested using 'Ireland' to avoid confusion with the closed channels, but maybe that's not necessary.

**Comment by freearhey** [comment]
I added a country suffix to the channel name: https://github.com/iptv-org/database/commit/02d856414ff6a2b3c3d82e72d432fe44f56758db

File: sites/sky.com/sky.com.channels.xml [hunk 1]
```diff
@@ -222,6 +222,8 @@
     <channel lang="en" xmltv_id="PopMax.uk" site_id="4262">Pop Max</channel>
     <channel lang="en" xmltv_id="PopMaxPlus1.uk" site_id="4263">POP Max+1</channel>
     <channel lang="en" xmltv_id="PopPlus1.uk" site_id="4216">POP+1</channel>
+    <channel lang="en" xmltv_id="PremierSports1.ie" site_id="1633">Premier Sports 1</channel>
+    <channel lang="en" xmltv_id="PremierSports2.ie" site_id="1635">Premier Sports 2</channel>
```
Comment (freearhey) [inline sites/sky.com/sky.com.channels.xml:226]:
```suggestion
    <channel lang="en" xmltv_id="PremierSports1Ireland.ie" site_id="1633">Premier Sports 1 Ireland</channel>
    <channel lang="en" xmltv_id="PremierSports2Ireland.ie" site_id="1635">Premier Sports 2 Ireland</channel>
```

**Review by freearhey — CHANGES_REQUESTED** [review]

**Review by freearhey — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Pull request merged**